extensions/amp-install-serviceworker/0.1/amp-install-serviceworker.js

```diff
@@ -103,7 +103,7 @@ export class AmpInstallServiceWorker extends AMP.BaseElement {
       // only engaged with superficially.
       timerFor(this.win).delay(() => {
         this.deferMutate(this.insertIframe_.bind(this));
-      }, 20000);
+      }, 10000);
```

Comment:
Would it be helpful to add an "if you change this, then change that" comment? Or would that just be extra clutter because they'll know to change it when the test fails?